registration-processor/registration-processor-core/src/main/java/io/mosip/registration/processor/core/abstractverticle/StageHealthCheckHandler.java

```diff
@@ -308,11 +307,32 @@ public void dispSpaceHealthChecker(Promise<Status> promise) {
 	 * @param promise {@link Promise} instance from handler
 	 * @param vertx  {@link Vertx} instance
 	 */
-	public void senderHealthHandler(Promise<Status> promise, Vertx vertx, String address) {
+	public void senderHealthHandler(Promise<Status> promise, Vertx vertx, MosipEventBus eventBus, String address) {
 		try {
-			vertx.eventBus().send(address, HealthConstant.PING);
-			final JsonObject result = resultBuilder.create().add(HealthConstant.RESPONSE, HealthConstant.PING).build();
-			promise.complete(Status.OK(result));
+			eventBus.consumerHealthCheck((msg, handler) -> {

```

Comment:
As discussed the handler is not requried and instred of message lets take the health check dto directly